.github/workflows/checks-quick.yml

```
@@ -35,6 +35,17 @@ jobs:
         with:
           app-id: ${{ secrets.WORKFLOW_STOPPER_RUNNER_APP_ID }}
           app-key: ${{ secrets.WORKFLOW_STOPPER_RUNNER_APP_KEY }}
+  prdoc:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    container:
+      image: docker.io/paritytech/prdoc:v0.1.1
+    steps:
+      - uses: actions/checkout@6d193bf28034eafb982f37bd894289fe649468fc # v4.1.7
+      - name: prdoc check
+        id: required
+        run: RUST_LOG=info prdoc check
```

```suggestion
    steps:
      - uses: actions/checkout@6d193bf28034eafb982f37bd894289fe649468fc # v4.1.7
      - name: prdoc check
        id: required
        run: docker run -v $(pwd):/repo -e RUST_LOG=info docker.io/paritytech/prdoc:v0.1.1 check
      - name: Stop all workflows if failed
        if: ${{ failure() && steps.required.conclusion == 'failure' && !github.event.pull_request.head.repo.fork }}
        uses: ./.github/actions/workflow-stopper
        with:
          app-id: ${{ secrets.WORKFLOW_STOPPER_RUNNER_APP_ID }}
          app-key: ${{ secrets.WORKFLOW_STOPPER_RUNNER_APP_KEY }}
```
We can try doing it this way: run the docker container with prdoc inside of the runner. Also I added the step that actually stops all other workflows if this check has failed.

---

sounds good, thank you